dev.ps1

```
@@ -0,0 +1,92 @@
+
+# We use a small wrapper: on each restart we rebuild then launch.
+# Node's --watch-path lets us scope what triggers restarts.
+$watchPaths = @(
+    "--watch-path=src"
+)
+
+# Build the command
+# --watch restarts the *entry* process on file change.
+# We chain: build first, then start from dist/.
+$nodeBin = "node"
+$args = @(
```

PSScriptAnalyzer (1.24.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)